src/Database/Schema/PostgresSchemaDialect.php

```diff
@@ -642,6 +642,10 @@ public function createTableSql(TableSchema $schema, array $columns, array $const
         $content = array_merge($columns, $constraints);
         $content = implode(",\n", array_filter($content));
         $tableName = $this->_driver->quoteIdentifier($schema->name());
+        $dbSchema = $this->_driver->schema();
+        if ($dbSchema != 'public') {
+            $tableName = $this->_driver->quoteIdentifier($dbSchema) . '.' . $tableName;
```

Comment:
We could just do this unconditionally

Reply:
I didn't try. I was aiming to change the output as little as possible.